app/views/pages/hiring_fundraiser.html.haml

```diff
@@ -0,0 +1,120 @@
+#page-hiring-job.bg-backgroundColorA.pb-16
+  %header
+    .lg-container.c-shapes
+      = link_to hiring_page_path, class: "c-prominent-link" do
+        = graphical_icon "arrow-left"
+        Back to All Job Listings
+
+      .role
+        = graphical_icon "hiring-fundraiser", category: 'graphics', css_class: 'graphic'
+        .information
+          %h1.text-h0.mb-12 Head of Partnerships and Fundraising
+          .details
+            .remote-tag
+              = graphical_icon "stars"
+              100% remote
+            .salary Salary: £50k (~$68k, ~€60k)
+
+  %article
+    .sm-container
+      %p.text-p-2xlarge.mb-40
+        %strong Become Exercism's Head of Partnerships and Fundraising!
+        Are you someone who can both think strategically, and do the hard work of finding and closing sponsorship and partnership deals?
+        If so, this role might be perfect for you. Come work with an exceptional team and help fix programming education.
+
+      %h2.text-h2.mb-12 Summary
+      %p.text-p-xlarge.mb-16
+        Exercism is a not-for-profit organisation providing world-class programming education to anyone for free, through a mixture of self-directed learning and mentoring.
+        We have grown to nearly a million members, purely by word-of-mouth, and are recognised as one of the leading education platforms online.
+        Exercism is built by a mixture of our core team and thousands of volunteers.
+      %p.text-p-xlarge.mb-16
+        To take our next steps we need to grow our core team, and to achieve that we need to place ourselves on more solid financial ground.
+        We're looking for someone to lead this process, to work with the CEO to create a fundraising strategy, and then to execute it, raising funds through sponsorship and partnerships.
+        Our target is to generate £500k of revenue in the next 12 months.
+      %p.text-p-xlarge.mb-32
+        Our values of kindness, empathy and personal-growth are core to everything we do - as students, as a community, and as a core team.
+        We're building an exceptional team, full of ambitious, passionate people who excel at what they do, love their work, and who want to honour the commitment of our thousands of volunteers.
```

Comment:
"We're building an exceptional team" sounds like there is nobody there yet which isn't true especially when you count maintainers as the wider team, maybe better "We're expanding our exceptional team ..."